Model/rebelShip.php

```diff
@@ -4,6 +4,9 @@
 
 class RebelShip extends AbstractShip
 {
+    private $jediFactor;
```

Comment:
See note in AbstractShip about removing this.

Reply:
Done